pkg/certs/cert-inspection/certgraphapi/type_registry.go

```diff
@@ -0,0 +1,98 @@
+package certgraphapi
+
+import "fmt"
+
+type PKIRegistryInfo struct {
+	CertificateAuthorityBundles []PKIRegistryInClusterCABundle    `json:"certificateAuthorityBundles"`
```

Comment:
if you want these for in-cluster, name them in-cluster to let a different type tackle the on-disk ones.

Reply:
Lets rename `PKIRegistryInfo` into `ClusterPKIRegistryInfo` to differentiate it from on-disk certificates? We probably don't need to have different sets of on-disk registries for different topologies and platforms